src/base/memory_monitor_cl.cpp

```diff
@@ -24,7 +24,7 @@
 
 #include "memory_monitor_cl.hpp"
 
-int mmon_convert_module_name_to_index (char *module_name)
+int mmon_convert_module_name_to_index (const char *module_name)
```

Comment:
it's declaration has to be changed as well :)

Reply:
I'll check it.